servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/RoundRobinLoadBalancer.java

```diff
@@ -221,7 +222,7 @@ private static <R, C extends LoadBalancedConnection> long nextResubscribeTime(
         final long lower = config.healthCheckResubscribeLowerBound;
         final long upper = config.healthCheckResubscribeUpperBound;
         final long currentTime = config.executor.currentTime(NANOSECONDS);
-        final long result = currentTime + (lower == upper ? lower : ThreadLocalRandom.current().nextLong(lower, upper));
```

Comment:
other similar methods explicitly add + 1 on the bound in previous to your conversions. Here it is not the case so is this behavior ok (so that its now suddenly inclusive)

Reply:
I think so: the bounds are not clear on their inclusiveness but it doesn't make a lot of sense to me that the lower bound is inclusive and the upper is not.